build.sh

```diff
@@ -170,6 +171,15 @@ function docker_build_with_version {
     fi
   fi
 
+  # If a specific Dockerfile requires some specific build options (like capabilities),
+  # let them be included in the build command.
+  # The content of the .build-opts-<OS> will be used as part of the docker build command.
+  if [ -f "$build_args_file" ] ; then
+      build_opts_from_file="$(cat "$build_args_file")"
```

Comment:
This is a bit security issue. Who will check, that content is right? What about if there is security or vulnerability content and tests are going to be run? I am trying to be a bit paranoic. :)

Reply:
How is this file different from others? I mean, if you can put any malicious code to this file, you can put it also anywhere else in the codebase.